microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -135,45 +137,55 @@ Define the specific markets in which you plan to offer your extension.  The defa
+| **Privacy policy requirements** | Required | Indicate if your extension accesses, collects, or transmits any personal information.  Your extension might fail the certification step if you select **Yes** and you don't provide a `Privacy policy URL`. |
+| **Privacy policy URL** | - | A valid privacy policy URL to communicate how your extension follows privacy laws and regulations.  You're responsible for ensuring your extension follows privacy laws and regulations.  You're also responsible for providing a privacy policy URL if any personal information is being accessed, transmitted, or collected by your extension.  To determine if your extension requires a privacy policy, see [Microsoft Store App Developer Agreement](https://go.microsoft.com/fwlink/?linkid=528905) and [Developer policies for the Microsoft Edge Add-ons store](/legal/microsoft-edge/extensions/developer-policies). |
+| **Website URL** | - | A webpage that provides additional information about your extension.  The `Website URL` must point to a webpage on your own website, not the web listing for your extension in the [Microsoft Edge Add-ons website](https://microsoftedge.microsoft.com/addons/Microsoft-Edge-Extensions-Home).  The `Website URL` helps users learn more about your extension, its features, and any other relevant information. |
+| **Support contact details** | - | The URL to your support webpage, or the email address to contact your support team. |
+| **Mature content** | - | Checkbox to specify if your extension includes mature content.  Extension rating helps determine the appropriate age group of the target audience of your extension.  To help determine if your extension has mature content, see [Developer policies for the Microsoft Edge Add-ons store](/legal/microsoft-edge/extensions/developer-policies). |
 
-> [!IMPORTANT]
-> Your developer/organization name, website URL, and support contact details that you submitted during registration are displayed to users on the Microsoft Edge Add-ons website.
+Properties that are not indicated as Required are Optional.
+
+Click the **Save & Continue** button<!-- todo: link, or button? --> to continue to the **Store listings** section.
 
 
 <!-- ========================================================================== -->
 ## Step 5: Add Store listing details for your extension
 
-The information provided in the following section is displayed to users who review your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.  To list your extension in the store, the following details are required.
+The information provided in the following section is displayed to users who review your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.
+
+To list your extension in the store, the following details are required:<!-- todo: for each lang? -->
```

Comment:
should lead-in say "for each language"?  List is missing two required properties according to table below.  Are all 4 properties required for at least one language?

live:
https://learn.microsoft.com/microsoft-edge/extensions-chromium/publish/publish-extension#step-5-add-store-listing-details-for-your-extension

PR rendered section:
https://review.learn.microsoft.com/microsoft-edge/extensions-chromium/publish/publish-extension?branch=pr-en-us-3330#step-5-add-store-listing-details-for-your-extension

Reply:
this "required" section seems to add confusion bc it is incomplete and is redundant with info in the table below.  I might try merging this paragraph/list into the table, and design the section lead-in to cover the point.